.idea/SIUE-Dixon-Fat-Segmentation-Algorithm.iml

```diff
@@ -2,7 +2,7 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Python 3.5.4 (D:\Users\addis\Anaconda3\python.exe)" jdkType="Python SDK" />
```

Comment:
So this line contains computer-specific information. This path will be different for each person. 

The fix for this will be to delete this file, commit it and then we add to gitignore to ignore this file. If you can't do this, then that's fine, I'll do it when I get a chance.